newsroom/ui_config_async.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional, Dict, Any
+from superdesk.core.module import Module
+from superdesk.core.mongo import MongoResourceConfig
+from superdesk.core.resources import ResourceModel, ResourceModelConfig
+from superdesk.core.resources.service import AsyncResourceService
+from content_api import MONGO_PREFIX
+
+
+class UiConfig(ResourceModel):
+    _id: str
+    preview: Optional[Dict[str, Any]] = None
+    details: Optional[Dict[str, Any]] = None
+    list: Optional[Dict[str, Any]] = None
+    advanced_search_tabs: Optional[Dict[str, Any]] = None
+    multi_select_topics: Optional[bool] = None
+    search: Optional[bool] = None
+    enable_global_topics: Optional[bool] = None
```

Comment:
Please provide meaningful defaults, especially for booleans. For example, the old `UIConfigResource` sets `enable_global_topics` to `True` as default